### ISSUE TITLE
Log Collection Using VSphere Elastic Agent Integration

### ISSUE THREAD

**Issue by sanjaruzic** [elastic/integrations#9190]
- I have a few questions about the vSphere Integration, more specifically about collecting the [logs part](https://docs.elastic.co/integrations/vsphere#logs) 

- From our documentation:
1. _This integration periodically fetches logs and metrics from [vSphere](https://www.vmware.com/products/vsphere.html) vCenter servers._
       This is a bit confusing since we create a listener on our side, so it's more about receiving than fetching

2.  _To access the logs, from the Kibana UI, you have to specify the network-accessible IP address of the host where the Elastic Agent will be deployed._
     (connected to the point 1.) - In this case, the user would have to configure syslog forwarding to the `IP:port `of the machine the Elastic Agent is installed on and that was configured in the integration. But this is not quite clear in the documentation, so it would be great if we could add more information here?
    
3.  Also, I'm not an expert in this area, but this seems to be a way to enable the log forwarding from the vSphere: [https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcsa.doc/GUID-9633A961-A5C3-4658-B099-B81E0512DC21.html](https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcsa.doc/GUID-9633A961-A5C3-4658-B099-B81E0512DC21.html)

- We have the following field defined that mentions alerts:
`event.kind` _gives high-level information about what type of information the event contains, without being specific to the contents of the event. For example, values of this field distinguish alert events from metric events._
 
But I'm not sure what alerts mean in this context and if it is possible to include [alarms data](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.monitoring.doc/GUID-9272E3B2-6A7F-427B-994C-B15FF8CADC25.html) in vSphere syslog and if this is the case, if we can parse them or not with our integration.

**Comment by jvalente-salemstate** [comment]
I agree. I am actually in the process of getting system configurations ready so I can enable a few integrations, including vSphere. The need for syslog for the logs data stream isn't apparent from this, meaning someone would need to go back and get the syslog forwarding working once they go to enable the integrations. It would be good to know this beforehand.

Looking at the sample logs and what's being parsed, some documentation on what needs to be enabled in vSphere / ESXi would be great too. It seems there is syslog from the ESXi hosts and vCenter itself. The latter is covered in the link you shared but remote syslog needs to be set up in the [ESXi configuration](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.esxi.upgrade.doc/GUID-8981F5FA-BB2A-47FB-A59A-7FC5C523CFDE.html#GUID-8981F5FA-BB2A-47FB-A59A-7FC5C523CFDE).

**Comment by alaudazzi** [comment]
@sanjaruzic 

> Looking at the sample logs and what's being parsed, some documentation on what needs to be enabled in vSphere / ESXi would be great too

Is this still required? If yes, would you mind providing the required input? The doc team is happy to help and review.

**Comment by alaudazzi** [comment]
After checking with Support, there have been no other cases regarding this integration configuration since then. We can create a KB article within support if it comes up again. Closing for now.
